core/src/main/java/org/apache/calcite/sql/SqlInsert.java

```diff
@@ -99,6 +100,12 @@ public final boolean isUpsert() {
     return getModifierNode(SqlInsertKeyword.UPSERT) != null;
   }
 
+  /** Returns whether this insert contains an OVERWRITE clause.
+   */
+  public final boolean isOverWrite() {
```

Comment:
Nitpick: I prefer `isOverwrite`. I think of it as one word and not two.

Reply:
done